docs/getting_gn.md

```diff
@@ -14,15 +14,27 @@ You can download the latest version of GN binary for Linux and macOS from Google
 
 Alternatively, you can build GN from source with a C++17 compiler:
 
+**macOS:**
+```bash
+git clone https://gn.googlesource.com/gn
+cd gn
+python3 build/gen.py
```

Comment:
This instruction is from https://gn.googlesource.com/gn/, so I prefer not to change it.